hack/test-end-to-end.sh

```diff
@@ -261,8 +258,13 @@ osc process -n test -f examples/sample-app/application-template-stibuild.json >
 osc process -n docker -f examples/sample-app/application-template-dockerbuild.json > "${DOCKER_CONFIG_FILE}"
 osc process -n custom -f examples/sample-app/application-template-custombuild.json > "${CUSTOM_CONFIG_FILE}"
 
+# Client setup (log in as e2e-user and set 'test' as the default project)
+echo "[INFO] Logging in as a regular user (e2e-user:pass) with project 'test'..."
+osc login -u e2e-user -p pass
```

Comment:
This will already create a new context, with pointers to the existing `cluster` and `namespace` (if present), but with a new `user` entry (with the token returned by the username/pass authentication).

The config file will look like this after this command (and allows me to switch between admin and e2e-user with `osc project master` and `osc project e2e-user` respectively):

```
apiVersion: v1
clusters:
- cluster:
    certificate-authority: /path/to/ca.crt
    server: https://10.97.6.110:8443
  name: master
contexts:
- context:
    cluster: master
    user: e2e-user
  name: e2e-user
- context:
    cluster: master
    user: admin
  name: master
current-context: e2e-user
kind: Config
preferences: {}
users:
- name: admin
  user:
    client-certificate: /path/to/cert.crt
    client-key-data: /path/to/key.key
- name: e2e-user
  user:
    token: OTNiODI3MmQtZTM5YS00MzU4LThmNTAtYmExYjQwNjlkMGE2
```